core/lib/spree/testing_support/dummy_app/rake_tasks.rb

```diff
@@ -19,6 +19,41 @@ def initialize(gem_root:, lib_name:)
   end
 end
 
+if Object.const_defined?(:SolidusAdmin)
+  namespace :solidus_admin do
+    namespace :tailwindcss do
+      desc 'Build Tailwind CSS'
+      task build: :dummy_environment do
+        require "solidus_admin"
+        require "tailwindcss/commands"
+
+        tailwindcss = Tailwindcss::Commands.executable
+
+        tailwindcss_command = [
+          tailwindcss,
+          "--config", SolidusAdmin::Engine.root.join("config/tailwind.config.js"),
+          "--input", SolidusAdmin::Engine.root.join("app/assets/stylesheets/solidus_admin/application.tailwind.css"),
+          "--output", DummyApp::Application.root.join("assets/builds/solidus_admin/tailwind.css"),
+        ]
```

Comment:
Note to self: We need to add the `lib/components` folder of the extension into the `--content`